content/inheritance/exercises/_index.md

```diff
@@ -0,0 +1,287 @@
+---
+title: "Exercises: Inheritance"
+date: 2023-01-31T15:39:38-06:00
+draft: false
+weight: 8
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Work on these exercises in a new Visual Studio .NET Core console app. Name your solution `Technology` and make a project inside of it with the same name.
+
+## Class Design
+
+Create three classes inside your solution: `Computer`, `Laptop`, and `SmartPhone`
+
+Before you start coding anything inside these classes, diagram how the three classes are going to be related to each other.  You can start with the diagram below.  Be sure to show how the base class extends each subclass.
+
+{{< mermaid >}}
+classDiagram
+  Computer -- Smartphone
+  Computer -- Laptop
+{{< /mermaid >}}
+
+{{% expand "Check your solution" %}}
```

Comment:
Please tab the Check your solution to match the location of the ones in the ordered list below.

Reply:
done.